data_pump/main.data_pump.py

```diff
@@ -0,0 +1,102 @@
+import logging
+
+import var_declarations as var
+from data_pump.bitstream import import_bitstream
+from data_pump.bitstreamformatregistry import import_bitstreamformatregistry
+from data_pump.bundle import import_bundle
+from data_pump.collection import import_collection
+from data_pump.community import import_community
+from data_pump.user_metadata import import_user_metadata
+from data_pump.eperson import import_eperson, import_group2eperson
+from data_pump.epersongroup import import_epersongroup, import_group2group
+from data_pump.handle import Handle
+from data_pump.item import import_item
+from data_pump.license import import_license
+from data_pump.metadata import Metadata
+from data_pump.registrationdata import import_registrationdata
+from data_pump.tasklistitem import import_tasklistitem
+from data_pump.user_registration import import_user_registration
+from utils import read_json
+
+
+def at_the_end_of_import(handle_class, statistics_dict):
```

Comment:
handle_class has the same name as handle_class from upper scope (method main). IDE complains. Rename.